test/framework/kubernetes.go

```diff
@@ -110,6 +110,11 @@ func GetPods(namespace string) (*corev1.PodList, error) {
 	return pods, nil
 }
 
+// GetPodsWithDeploymentConfig retrieves pods with a deploymentconfig label set to <dcName>
+func GetPodsWithDeploymentConfig(namespace string, dcName string) (*corev1.PodList, error) {
```

Comment:
We're getting pods with label "deploymentConfig". For me, it matches better with "WithDeploymentConfig", or "WithDeploymentConfigLabel" maybe. What do you think?

Reply:
IMHO the `GetPodsForDeploymentConfig` sounds better. The fact that we use deployment config label is implementation detail and is not related to the function signature. Here we just want to return pods which are related to this deployment config.

Reply:
or maybe just write a simple method `GetPodsWithLabel(namespace, labelKey, labelValue string)` wdyt ?
and `"deploymentconfig"` could be var ?

Reply:
What I wanted to avoid is to have the "deploymentconfig" hard coded in several places.
GetPodsForDeploymentConfig makes me think that the pods are for DeploymentConfig and it's actually the other way around. What about GetPodsByDeploymentConfig (or GetPodsByDeploymentConfigLabel) ?

Reply:
`GetPodsByDeploymentConfig` seems good to me